### PR TITLE
[DL-255]: fixed entire section to match FIGMA comp

### PR DESCRIPTION
**Pull request by jmesches-fairway**

### CODE REVIEW

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss [hunk 1]
```diff
@@ -3,47 +3,56 @@
...
-    font-size: $font-size-3;
-    font-weight: 500;
-    line-height: 133%;
-  }
+.background-container {
 
-  .leading-icon {
+    padding-top: $spacer;
+    margin-bottom: $spacer-lg;
+    margin-left: 2rem;
```
Comment (jdbernard) [inline webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Can we use `$spacer-lg` here?

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss [hunk 2]
```diff
@@ -3,47 +3,56 @@
...
-    i, svg {
-      color: $success-900;
-      background: url('/assets/images/green-spot-3.png') no-repeat;
-      background-size: contain;
-      padding: 7px;
-      margin-bottom: -7px;
-      margin-left: -16px;
-      font-size: 1.75rem;
+    h2 {
+      font-size: 1.76875rem;
```
Comment (jdbernard) [inline webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Can we use `$font-size-4` here?

File: webapp/src/app/resource/playlist/background/playlist-background.component.html [hunk 3]
```diff
@@ -1,11 +1,13 @@
-<section #sectionHeader *ngIf="overview">
+<article  class="background-container" *ngIf="overview">
   <h2 class="leading-icon">
     <i class="fas fa-check-circle"></i>
     IAB Background
     <sbdl-tooltip text="A description of the interim assessment connected with this playlist, including a brief summary of what students should know or be able to do.">
       <sbdl-icon icon="info"></sbdl-icon>
     </sbdl-tooltip>
   </h2>
+<section #sectionHeader class="background-table">
```
Comment (jdbernard) [inline webapp/src/app/resource/playlist/background/playlist-background.component.html]: Can we move `#sectionHeader` to either the new `article` element or the `h2.leading-icon` element? The element tagged with `#sectionHeader` gets used as a scroll target when the user clicks on this section in the outline. We want it to drop them at the top of the section.

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 1 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Probably. I didn't change any of this... just reformatted for the most part

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 2 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Same as above

File: webapp/src/app/resource/playlist/background/playlist-background.component.html (hunk 3 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.html]: Yup, I noticed this while tackling the next ticket. I'll add that change to this PR, too

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 2 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Actually, realized I screwed this up! Fixed and adding to the PR

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 1 above)
Comment (jdbernard) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: OK, thanks!

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 2 above)
Comment (jdbernard) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: No worries. The font sizing magic numbers threw me at first as well. They're based on ratios off the base font size and come originally from the Material design standard we were consulting when designing this. I started replacing the sizes with variables but not surprised I missed some spots. Thanks!

File: webapp/src/app/resource/playlist/background/playlist-background.component.html (hunk 3 above)
Comment (jdbernard) [reply webapp/src/app/resource/playlist/background/playlist-background.component.html]: Sounds good. Thanks for jumping in and hammering some of this out.

File: webapp/src/app/resource/playlist/background/playlist-background.component.scss (hunk 2 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.scss]: Fixed

File: webapp/src/app/resource/playlist/background/playlist-background.component.html (hunk 3 above)
Comment (jmesches-fairway) [reply webapp/src/app/resource/playlist/background/playlist-background.component.html]: Fixed

**Pull request merged**